### DISCUSSION TITLE
Working with Aergolite - how to do initial setup

### DISCUSSION THREAD

**Discussion by tutiplain** [aergoio/aergolite#7] [Q&A] [answered]
Hi all,

  I am trying to get aergolite to work, but I'm afraid I'm having difficulty understanding the instructions on the README. Right now, I am working with the "sqlite3" command shell just to get it working, before moving on to using aergolite in an application. I managed to compile the repo without difficulty.  I run the "sqlite3" command on my terminal and then I run the following command:

`.open "file=test.db?blockchain=on&password=test`

I can then run "PRAGMA blockchain_status;" and can see that there is an unauthorized node reported. I try to authorize the node using: 

`PRAGMA add_node="pubKey"`

For the public key I use the value reported as the pubKey from the previous blockchain_status command. However, this simply returns "Error: signature failed".  Am I not doing this process correctly? Also, where are the keys stored? I see no new files in my directory other than those created by aergolite (-consensus, -local, -wal, -shm, etc.). Is the first node created not the blockchain administrator? Any help will be greatly appreciated. Thanks in anticipation.

**Comment by kroggen** [comment, marked as answer]
Please read this section:

[Signing transactions](https://github.com/aergoio/aergolite#signing-transactions)

You need to either use a Ledger Nano S or implement the code to sign transactions on your Go app. Only the first option works with the `sqlite3` shell. Notice that the sample code for Go does not include code for signing a transaction, you need to implement it yourself, using elliptic curve secp256k1.


Then later you can check these sections:

[Blockchain Administrator](https://github.com/aergoio/aergolite#blockchain-administrator)

[Node Discovery](https://github.com/aergoio/aergolite#node-discovery)

And these [Bootstrapping instructions](https://github.com/aergoio/aergolite/wiki/Bootstrapping#managing-the-private-key-yourself)
